package.json

```diff
@@ -16,7 +16,7 @@
 		"test:watch": "jest --watch",
 		"type-check": "tsc --noEmit",
 		"type-check:watch": "npm run type-check -- --watch",
-		"lint": "eslint --ext .js,.jsx,.ts,.tsx --resolve-plugins-relative-to node_modules/@zextras/carbonio-ui-configs src",
+		"lint": "eslint --ext .js,.jsx,.ts,.tsx --quiet --resolve-plugins-relative-to node_modules/@zextras/carbonio-ui-configs src",
```

Comment:
i disagree. the --quiet flag suppresses warnings only, I really don't see the risk in that. 
the same implementation is used in mails module

Reply:
I agree with Luca. I believe it should always show everything by default and then if there is an error you can manually add the quiet option. We need to be aware of all the warnings in our projects as they can be important. Most of them are just warnings because we can't afford to work on it but they should become errors in the next future

Reply:
The risk is that, by hiding all the warnings, to not be aware of them and therefore to never resolve them.
I know that seeing hundreds of warnings is overwhelming, but I don't think that hiding them is the proper solution.
I notice that also Mails has this configuration so IMO we should change that too